forgerock-auth/src/main/java/org/forgerock/android/auth/ConfigHelper.kt

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) 2022 ForgeRock. All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+package org.forgerock.android.auth
+
+import android.content.Context
+import android.content.SharedPreferences
+
+interface ConfigInterface {
+    fun loadFromCache(context: Context?): FROptions
+}
+
+class ConfigHelper: ConfigInterface {
+
+    companion object {
+
+        const val ORG_FORGEROCK_V_1_HOSTS = "org.forgerock.v1.HOSTS"
+
+        @JvmStatic
+        fun cache(context: Context?, frOptions: FROptions) {
+            context?.let {
+                val sharedPreferences =
+                    it.getSharedPreferences(ORG_FORGEROCK_V_1_HOSTS, Context.MODE_PRIVATE)
+                sharedPreferences.edit().putString("url", frOptions.server.url)
+                    .putString("realm", frOptions.server.realm)
+                    .putString("cookieName", frOptions.server.cookieName)
+                    .putString("client_id", frOptions.oauth.oauthClientId)
+                    .putString("revoke_endpoint", frOptions.urlPath.revokeEndpoint)
+                    .putString("end_session_endpoint", frOptions.urlPath.endSessionEndpoint)
```

Comment:
U need to call apply()

Reply:
Done , sorry stoyan pulled me for something, forgot to commit